crates/bevy_mod_scripting_core/src/asset.rs

```diff
@@ -18,6 +18,7 @@ use bevy::{
     reflect::TypePath,
     utils::HashMap,
 };
+use mlua::Table;
```

Comment:
🙅 This is introducing lua logic into the bms_core layer. The idea is to only keep language agnostic logic in here